source/beamline/BioXAS/BioXASZebraPulseControl.cpp

```diff
@@ -200,8 +203,10 @@ void BioXASZebraPulseControl::setInputValue(int value)
 
 void BioXASZebraPulseControl::setEdgeTriggerValue(int value)
 {
-	if (!edgeTriggerControl_->withinTolerance(double(value)))
-		edgeTriggerControl_->move(double(value));
+	if (edgeTriggerValue_ != value) {
+		edgeTriggerValue_ = value;
```

Comment:
Perhaps instead of calling this value edgeTriggerValue_, a better name would be edgeTriggerInitialValue_ or edgeTriggerValuePreference_?